docker-composer.env

```diff
@@ -0,0 +1 @@
+7.2-1.8.5
```

Comment:
Dit bestand kent ook niemand, moeten deze en de docker-compose.yml niet in de .gitignore?